erpnext/hr/doctype/daily_work_summary/daily_work_summary.py

```diff
@@ -52,6 +52,18 @@ def get_message_details(self):
 			d.sender_name = emp[0].employee_name if emp else d.sender
 			d.image = emp[0].image if emp and emp[0].image else None
 
+			# make thumbnail image
+			try:
+				if d.image:
+					file_name = frappe.get_list('File', {'file_url': d.image})
+
+					if file_name:
+						file_name = file_name[0].name
+						file_doc = frappe.get_doc('File', file_name)
+						d.image = file_doc.make_thumbnail(set_as_thumbnail=False, width=100, height=100)
```

Comment:
aspect ratio?

Also `make_thumbnail` will always make a new thumbnail. Make a new method `get_thumbnail_url` that will make a thumbnail if missing...

Reply:
The thumbnail created here is only for the email, `get_thumbnail_url` would return the default thumbnail of 300x300